### ISSUE TITLE
Private NuGet Feeds

### ISSUE THREAD

**Issue by fizxmike** [dotnet/docs#5340]
I get authorization error if I try add package from a private feed:
```
dotnet add package SomePkg --source https://myvsaccount.pkgs.visualstudio.com/_packaging/SomePkg/nuget/v3/index.json
[...]
error: Unable to load the service index for source https://myvsaccount.pkgs.visualstudio.com/_packaging/SomePkg/nuget/v3/index.json.
error:   Response status code does not indicate success: 401 (Unauthorized).
```

I would expect the VisualStudio.com login window to pop up (like it does when I "git clone" from the command line for the same project) and prompt me for credentials...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff30b638-44e3-077c-3cad-b61d1314c303
* Version Independent ID: 784413e1-77f5-f2e6-a8f4-56a73fe2d3bf
* Content: [dotnet add package command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package)
* Content Source: [docs/core/tools/dotnet-add-package.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-add-package.md)
* Service: **dotnet-core**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

**Comment by fizxmike** [comment]
This seems to help:
https://docs.microsoft.com/en-us/vsts/package/nuget/dotnet-exe?view=vsts

... but PAC in a file that is routinely checked in to version control is a bad pattern and begs for disaster... how does git interface with VSTS so seamlessly, but dotnet CLI does not?

... any suggestions?

**Comment by mairaw** [comment]
@KathleenDollard @livarcocc who can help with this question about using private feeds with the dotnet add package command?

**Comment by livarcocc** [comment]
@rohit21agrawal

**Comment by mairaw** [comment]
@rohit21agrawal can you help here?

**Comment by nkolev92** [comment]
dotnet.exe currently does not authentication support for NuGet feeds. 

We're working on adding that in this [issue](https://github.com/NuGet/Home/issues/6486).

It's currently scheduled to ship with Visual Studio 15.8.

**Comment by BillWagner** [comment]
closed as duplicate of [NuGet/Home/#6486](https://github.com/NuGet/Home/issues/6486)
